docs/code_ref/coordinates/carrington.rst

```diff
@@ -0,0 +1,126 @@
+.. _sunpy-coordinates-carrington:
+
+Calculating Carrington longitude
+================================
+
+Carrington coordinates are a heliographic coordinate system, where the longitude is determined assuming that the Sun has rotated at a sidereal period of ~25.38 days starting at zero longitude at a specific time on 1853 November 9.
+For a given observer at a given observation time, the center of the disk of the Sun as seen by that observer (also known as the apparent sub-observer point) has a Carrington latitude and longitude.
+When the observer is at Earth, the apparent sub-Earth Carrington latitude and longitude are also known as |B0| and |L0|, respectively.
```

Comment:
> * Why we said here "apparent"?

The apparent sub-observer point is different from the true sub-observer point because the observer is seeing light from the past.  For example, the apparent center of the disk for an observer at Earth is ~500 seconds in the past; whatever the observer sees at disk center is no longer actually there in "true" coordinates.

> * What does "sub" in sub-Earth means? Should it be "sun-Earth"?

"sub-Earth" is a special case of "sub-observer".  "sub-observer" point means the point on the surface directly "below" the observer.

Reply:
Re. "aparent": Gotya! I understood that later on in the text. I wonder if that could have this comment as a footnote. 
Re. "sub-Earth": Aaahhh, I didn't know that. Is that for the Earth a constant value or there are variations from observatories at higher mountains and at different places on Earth? Maybe a foot-note with that definition there could be also helpful. 

Maybe I've forgotten all that I knew about observatories and observations... so maybe it's just me and it's common knowledge across the community. If that's the case, then we don't need to define these terms. :grin:

Reply:
I added a footnote

Reply:
I forgot to add a footnote for "sub-Earth".

Sub-Earth, without any qualification, means the center of the Earth.  Of course, ground-based observatories aren't at the center of the Earth, so there will be some very small differences in their sub-observer longitudes.

Reply:
Footnote added